pkg/application/options.go

```diff
@@ -158,122 +152,89 @@ func WithKernelSettingsFromConfig(app *App) {
 }
 
 func WithLoggerApplicationTag(app *App) {
-	app.addLoggerOption(func(config cfg.GosoConf, logger mon.GosoLog) error {
+	app.addLoggerOption(func(config cfg.GosoConf, logger log.GosoLogger) error {
 		if !config.IsSet("app_name") {
 			return errors.New("can not get application name from config to set it on logger")
 		}
 
-		return logger.Option(mon.WithTags(map[string]interface{}{
+		return logger.Option(log.WithFields(map[string]interface{}{
 			"application": config.GetString("app_name"),
 		}))
 	})
 }
 
-func WithLoggerContextFieldsMessageEncoder() Option {
-	return func(app *App) {
-		app.addLoggerOption(func(config cfg.GosoConf, logger mon.GosoLog) error {
-			stream.AddDefaultEncodeHandler(mon.NewMessageWithLoggingFieldsEncoder(config, logger))
-			return nil
-		})
-	}
+func WithLoggerContextFieldsMessageEncoder(app *App) {
+	app.addLoggerOption(func(config cfg.GosoConf, logger log.GosoLogger) error {
+		stream.AddDefaultEncodeHandler(log.NewMessageWithLoggingFieldsEncoder(config, logger))
+		return nil
+	})
 }
 
-func WithLoggerContextFieldsResolver(resolver ...mon.ContextFieldsResolver) Option {
+func WithLoggerContextFieldsResolver(resolver ...log.ContextFieldsResolver) Option {
 	return func(app *App) {
-		app.addLoggerOption(func(config cfg.GosoConf, logger mon.GosoLog) error {
-			return logger.Option(mon.WithContextFieldsResolver(resolver...))
+		app.addLoggerOption(func(config cfg.GosoConf, logger log.GosoLogger) error {
+			return logger.Option(log.WithContextFieldsResolver(resolver...))
 		})
 	}
 }
 
-func WithLoggerFormat(format string) Option {
+func WithLoggerHandlers(handler ...log.Handler) Option {
 	return func(app *App) {
-		app.addLoggerOption(func(config cfg.GosoConf, logger mon.GosoLog) error {
-			return logger.Option(mon.WithFormat(format))
+		app.addLoggerOption(func(config cfg.GosoConf, logger log.GosoLogger) error {
+			return logger.Option(log.WithHandlers(handler...))
 		})
 	}
 }
 
-func WithLoggerHook(hook mon.LoggerHook) Option {
-	return func(app *App) {
-		app.addLoggerOption(func(config cfg.GosoConf, logger mon.GosoLog) error {
-			return logger.Option(mon.WithHook(hook))
-		})
-	}
-}
+func WithLoggerHandlersFromConfig(app *App) {
+	app.addLoggerOption(func(config cfg.GosoConf, logger log.GosoLogger) error {
+		var err error
+		var handlers []log.Handler
 
-func WithLoggerLevel(level string) Option {
-	return func(app *App) {
-		app.addLoggerOption(func(config cfg.GosoConf, logger mon.GosoLog) error {
-			return logger.Option(mon.WithLevel(level))
-		})
-	}
-}
+		if handlers, err = log.NewHandlersFromConfig(config); err != nil {
+			return fmt.Errorf("can not create handlers from config: %w", err)
+		}
 
-func WithLoggerMetricHook(app *App) {
-	app.addLoggerOption(func(config cfg.GosoConf, logger mon.GosoLog) error {
-		metricHook := mon.NewMetricHook()
-		return logger.Option(mon.WithHook(metricHook))
+		if len(handlers) == 0 {
+			handlers = append(handlers, log.NewCliHandler())
```

Comment:
It's the old and new default logger behavior, yes

Reply:
Can we name it as such? :D

Reply:
I would rather keep it like it is..  
Imagine a second builtin handler like `NewJsonHandler`..  
Just because one of these is used as the default, I don't have to name it DefaultHandler..  
For me it's more obvious, that the cli format is the default format..

Reply:
obsolete due to named handlers